package.json

```
@@ -8,9 +8,9 @@
 		"lint": "eslint ./",
 		"lint-fix": "npm run lint -- --fix",
 		"lint-diff": "LIST=`git diff-index --name-only HEAD | grep '.*\\.js$';`; if [ \"$LIST\" ]; then eslint $LIST; fi",
-		"precommit-msg": "echo 'Running pre-commit checks... (skip using --no-verify)' && exit 0",
+		"precommit-msg": "echo 'Running pre-commit checks... (skip using --no-verify)'",
 		"semantic-release": "semantic-release",
-		"test": "echo 'No tests' && exit 0"
+		"test": "echo 'implement tests' && exit 1"
```

Unfortunately `package-template` still has a requirement to pass tests, to do things like tag it's own releases and update it's own dependencies. Either we revert this change or fudge the CI test runner job.

```suggestion
		"test": "echo 'implement tests' && exit 0"
```

---

ok will change 😢